build.sbt

```diff
@@ -154,27 +166,25 @@ lazy val `log-collector` = project
       dependencies.kudu,
       dependencies.kuduBinary,
       dependencies.kuduTestUtils
-    )
+    ) ++ dependencies.solr ++ dependencies.common
   )
-  .dependsOn(`test-common` % "test,it")
   .dependsOn(common)
+  .dependsOn(solrModule)
+  .dependsOn(solrModule % "test->test")
 
 lazy val `collection-roller` = project
-  .configs(IntegrationTest)
-  .settings(Defaults.itSettings: _*)
   .settings(
     name := "collection-roller",
     mainClass in Compile := Some("io.phdata.pulse.logcollector.roller.CollectionRollerMain"),
     settings,
     libraryDependencies ++= dependencies.common
-    ++ Seq(dependencies.scalaYaml, dependencies.scallop)
+    ++ Seq(dependencies.scalaYaml, dependencies.scallop) ++ dependencies.solr
```

Comment:
I'm not sure why the solr dependencies aren't added transitively throught the dependency on the Pulse solr module, but they aren't.